posix.mak

```diff
@@ -370,6 +372,13 @@ unittest/%.run : $(ROOT)/unittest/test_runner
 	    RET=$$? ; rm -rf $$T ; exit $$RET                                                                   \
 	  )
 
+# Target for quickly debugging a single module
+# For example: DEBUGGER=ddd BUILD=debug make -f posix.mak std/format.debug
+# ddd in this case is a graphical frontend to gdb
+%.debug : %.d $(LIB)
+	$(DMD) $(DFLAGS) -debug -main -unittest $(LIB) -defaultlib= -debuglib= $(LINKDL) $< &&     \
+	$(DEBUGGER) ./$(basename $(notdir $<))
```

Comment:
It would also be nice to write the binary and object file to a temporary location and do a cleanup after the debugger ran.